create-neon/src/bin/create-neon.ts

```diff
@@ -40,7 +46,9 @@ async function main(name: string) {
 
   try {
     pkg = await Package.create(name);
+
   } catch (err) {
+    await fs.rm(process.argv[2], { recursive: true, force: true })
```

Comment:
This should use `name` instead of the `argv` directly.

Also, I think it would be better to move the `fs.rm` into `die`.